systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture.url }}" class="profile-pic img-square img-responsive img-wh-100" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
+    <img src="{% static 'img/default.png' %}" class="profile-pic img-square img-responsive img-wh-100"
+    alt="{{ user }} profile picture"/>
+  </a>
+{% endif %}
+<a href="{{ user.get_absolute_url }}">
+  <p class="text-indent35"> {{ user }} </p>
```

Comment:
So to remove the indent and make it look nice, you need to center both the image with `margin: 0 auto;` and the text with `text-align: center;`